src/components/AddClient/index.js

```diff
@@ -54,6 +57,7 @@ export function AddClient({ setIsFormShowing, clientsData, setClientsData }) {
                 realEstatesNeighborhood: [],
                 budget: "",
                 additionalNotes: "",
+                userId: "",
```

Comment:
You could create a function for getting the usedId. 
```suggestion
                userId: getAdminUserId(),
```